OpcUaClient/OpcUaClient.hpp

```diff
@@ -112,6 +120,47 @@ class OpcUaClient : public Dashboard::IDashboardDataClient {
 
   Subscription m_subscr;
 
+  const std::map<std::string, size_t> XMLtoUaType={
+				{ "opc:Boolean", UA_TYPES_BOOLEAN },
+				{ "opc:SByte", UA_TYPES_SBYTE },
+				{ "opc:Byte", UA_TYPES_BYTE },
+				{ "opc:Int16", UA_TYPES_INT16 },
+				{ "opc:UInt16", UA_TYPES_UINT16 },
+				{ "opc:Int32", UA_TYPES_INT32 },
+				{ "opc:UInt32", UA_TYPES_UINT32 },
+				{ "opc:Int64", UA_TYPES_INT64 },
+				{ "opc:UInt64", UA_TYPES_UINT64 },
+				{ "opc:Float", UA_TYPES_FLOAT },
+				{ "opc:Double", UA_TYPES_DOUBLE },
+				{ "opc:CharArray", UA_TYPES_STRING },
+				{ "opc:DateTime", UA_TYPES_DATETIME },
+				{ "opc:Guid", UA_TYPES_GUID },
+				{ "opc:ByteString", UA_TYPES_BYTESTRING },
+				{ "opc:String", UA_TYPES_STRING },
+				{ "ua:XmlElement", UA_TYPES_XMLELEMENT },
+				{ "ua:NodeId", UA_TYPES_NODEID },
+				{ "ua:ExpandedNodeId", UA_TYPES_EXPANDEDNODEID },
+				{ "ua:StatusCode", UA_TYPES_STATUSCODE },
+				{ "ua:QualifiedName", UA_TYPES_QUALIFIEDNAME },
+				{ "ua:LocalizedText", UA_TYPES_LOCALIZEDTEXT },
+				{ "ua:ExtensionObject", UA_TYPES_EXTENSIONOBJECT },
+				{ "ua:DataValue", UA_TYPES_DATAVALUE },
+				{ "ua:Variant", UA_TYPES_VARIANT },
+				{ "ua:DiagnosticInfo", UA_TYPES_DIAGNOSTICINFO },
+				{ "ua:SimpleAttributeOperand", UA_TYPES_SIMPLEATTRIBUTEOPERAND},
+				{ "ua:ContentFilter", UA_TYPES_CONTENTFILTER},
+				{ "ua:Range", UA_TYPES_RANGE},
+				{ "ua:EUInformation", UA_TYPES_EUINFORMATION}
+
+				/* TODO ??? 
```

Comment:
What about these other Types?

Reply:
The aggregating Types are handled different. I suggest to leave it this way until we run into problems.